packages/block-editor/src/store/test/reducer.js

```diff
@@ -4505,5 +4505,296 @@ describe( 'state', () => {
 				);
 			} );
 		} );
+
+		// Tests for the contentsOnly experiments pattern.
```

Comment:
Nit: Is it worth tweaking `contentsOnly` to `contentOnly` for easier searching for all things contentOnly related? It could also be cleaned up when the experiment is stabilized.

Reply:
Oh yeah, typo, I've updated that and also addressed the other comment. Thanks for reviewing!

Reply:
Hmm, I accidentally did something weird with git, merged the PR assuming the extra commit had landed, but the push was rejected.

Guess I'm making another PR with those changes!

Reply:
All good, these were the tiniest of nits anyhow.